examples/cis5-smart-contract-wallet/src/lib.rs

```diff
@@ -792,10 +808,12 @@ fn withdraw_ccd(
                     Address::Account(account_address)
                 }
                 Receiver::Contract(contract_address, function) => {
+                    let receiving_contract_input_parameter = NoLengthPrependedBytes(data.as_ref());
+
                     // If the receiver is a contract: invoke the receive hook function.
                     host.invoke_contract(
                         &contract_address,
-                        &data,
+                        &receiving_contract_input_parameter,
```

Comment:
You can use [`host.invoke_contract_raw`](https://docs.rs/concordium-std/latest/concordium_std/trait.HasHost.html#tymethod.invoke_contract_raw) and pass `Parameter::new_unchecked(data.as_ref())` instead.
It would save allocating the bytes again and remove the need for `NoLengthPrependedBytes`